example/material_stepper_example/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: material_stepper_example
+author: Dart Team <misc@dartlang.org>
+environment:
+  sdk: '>=2.0.0-dev.56.0 <2.0.0'
+dependencies:
+  angular: 5.0.0-alpha+14
+  angular_components: 0.9.0-alpha+14
+  angular_gallery_section:
+    path: ../../angular_gallery_section
+  build_config: ^0.2.6
```

Comment:
Here and in other `pubspec.yaml` files you might want to upgrade to `build_config: ^0.3.0` (which, I believe, offers better error reporting).

Reply:
Thanks. I'll look into making sure we get updated to the latest.